File: src/Suframa.Sciex.UserInterface/www/app/view-model/IdentificacaoPessoaJuridicaVM.ts
import { pessoaJuridicaInscricaoEstadualVM } from './PessoaJuridicaInscricaoEstadualVM';
import { manterEnderecoVM } from './ManterEnderecoVM';
import { documentoComprobatorioVM } from './DocumentoComprobatorioVM';
import { FiltroCadastroPessoaJuridicaVM } from './FiltroCadastroPessoaJuridicaVM';

export class identificacaoPessoaJuridicaVM {
	cnpj: string;
	codigo: string;
	complemento: string;
	dataAlteracao?: Date;
	dataInclusao?: Date;
	dataRegistro?: Date;
	documentosComprobatorios: documentoComprobatorioVM[];
	email: string;
	endereco: manterEnderecoVM;
	filtroCadastroPessoaJuridica: FiltroCadastroPessoaJuridicaVM;
	idCep: number;
	idPessoaJuridica: number;
	idPorteEmpresa?: number;
	idRequerimento?: number;
	idTipoRequerimento: number;
	idUnidadeCadastradora: number;
	isCredenciamento: boolean;
	isCredenciamentoTransportador: boolean;
	isEntidadeEmpresarial: boolean;
	isQuadroSocietario: boolean;
	modalidadeTransportador: number;
	nomeFantasia: string;
	numeroEndereco: string;
	numeroInscricaoMunicipal: string;
	numeroRegistroConstituicao: string;
	pessoaJuridicaInscricaoEstadual: pessoaJuridicaInscricaoEstadualVM[];
	pontoReferencia: string;
	ramal?: number;
	razaoSocial: string;
	statusOptanteSimples?: boolean;
	telefone?: number;
	valorCapitalSocial?: number;
}

File: src/Suframa.Sciex.CrossCutting.DataTransferObject/ViewModel/IdentificacaoPessoaJuridicaVM.cs
using Suframa.Sciex.CrossCutting.DataTransferObject.Enum;
using System;
using System.Collections.Generic;
using System.Linq;

namespace Suframa.Sciex.CrossCutting.DataTransferObject.ViewModel
{
	public class IdentificacaoPessoaJuridicaVM
	{
		public CepVM Cep { get; set; }
		public string Cnpj { get; set; }
		public string Codigo { get; set; }
		public string Complemento { get; set; }
		public DateTime? DataAlteracao { get; set; }
		public DateTime? DataInclusao { get; set; }
		public DateTime? DataRegistro { get; set; }
		public IEnumerable<DocumentoComprobatorioVM> DocumentosComprobatorios { get; set; }
		public string Email { get; set; }
		public ManterEnderecoVM Endereco { get; set; }
		public FiltroCadastroPessoaJuridicaVM FiltroCadastroPessoaJuridica { get; set; }
		public int? IdCep { get; set; }
		public int? IdPessoaJuridica { get; set; }
		public int? IdPorteEmpresa { get; set; }
		public int? IdRequerimento { get; set; }
		public int? IdTipoRequerimento { get; set; }
		public int? IdUnidadeCadastradora { get; set; }
		public bool IsCredenciamento { get; set; }
		public bool IsCredenciamentoTransportador { get; set; }
		public bool IsEntidadeEmpresarial { get; set; }
		public bool IsQuadroSocietario { get; set; }
		public EnumTipoTransportador? ModalidadeTransportador { get; set; }
		public string NomeFantasia { get; set; }
		public string NumeroEndereco { get; set; }
		public string NumeroInscricaoMunicipal { get; set; }
		public string NumeroRegistroConstituicao { get; set; }
		public IEnumerable<PessoaJuridicaInscricaoEstadualVM> PessoaJuridicaInscricaoEstadual { get; set; } = Enumerable.Empty<PessoaJuridicaInscricaoEstadualVM>();
		public string PontoReferencia { get; set; }
		public decimal? Ramal { get; set; }
		public string RazaoSocial { get; set; }
		public bool? StatusOptanteSimples { get; set; }
		public decimal? Telefone { get; set; }
		public double? ValorCapitalSocial { get; set; }
	}
}